src/Template/Pages/home.ctp

```diff
@@ -170,6 +170,8 @@ $cakeDescription = 'CakePHP: the rapid development php framework';
                     <ul><li>Everything CakePHP</li></ul></li>
                     <li><a href="http://plugins.cakephp.org">CakePHP plugins repo</a>
                     <ul><li>A comprehensive list of all CakePHP plugins created by the community</li></ul></li>
+                    <li><a href="http://discourse.cakephp.org/">CakePHP Official Forum</a>
+                    <ul><li>CakePHP discussion forum</li></ul></li>
                     <li><a href="https://groups.google.com/group/cake-php">CakePHP Google Group</a>
```

Comment:
If its still useful it should come after the awesome list. If you consider it legacy we could mark it as such (Legacy).
Same goes for the CakePHP Google Group in case you want one single forum on discourse.cakephp.org.